cli/ext/build/ext.stub.php

```
@@ -0,0 +1,37 @@
+
+function emit_event(?array $userContext, array $event, string $from): int {}
+
+class Worker {
+
+    public function __construct() {}
+
+    public static function GetCurrent(): ?Worker {}
+
+    public function __destruct() {}
+
+    public function queryState(string $stateId): array {}
```

PHPMD (2.15.0)

Avoid unused parameters such as '$stateId'. (Unused Code Rules)

(UnusedFormalParameter)